pkg/api/helpers.go

```diff
@@ -73,3 +74,11 @@ var standardResources = util.NewStringSet(
 func IsStandardResourceName(str string) bool {
 	return standardResources.Has(str)
 }
+
+func ProbeTimeout(container Container) time.Duration {
```

Comment:
I agree. This is only used in one spot and should probably just be inlined until the defaults work is in, then we can just remove it.